src/webextension/list/manage/components/account-summary.css

```diff
@@ -124,18 +145,20 @@
   height: 16px;
   background-repeat: no-repeat;
   background-size: 16px 16px;
```

Comment:
If we're switching from `background-image` to `mask-image` below, we don't need these two lines anymore, right?